cmd/dump_repo.go

```diff
@@ -159,9 +160,14 @@ func runDumpRepository(ctx *cli.Context) error {
 		}
 	}
 
+	repoDir := ctx.String("repo_dir")
+	if dir, _ := os.ReadDir(repoDir); len(dir) > 0 {
```

Comment:
repoDir could be not exist or empty directory.

Reply:
If repoDir doesn't exist, the `len(dir)` is 0? Add more comments?

Reply:
Let's add a comment and move on